apps/server/src/modules/tldraw/repo/tldraw.repo.ts

```diff
@@ -14,9 +14,6 @@ export class TldrawRepo {
 
 	public async findByDocName(docName: string): Promise<TldrawDrawing[]> {
 		const domainObject = await this.em.find(TldrawDrawing, { docName });
```

Comment:
`domainObject` is not a good name for this variable. On the one hand it is an array (or more precisely a collection) and not a single object. On the other hand the objects returned here are not Domain Objects in our sense, but MikroORM-entities. `drawings` or `documents` sound like reasonable names to me. You may not change it now in this PR, but then please do in a forthcoming.

Reply:
You are right, i will surely change that now